doc/asciidoc/algorithms/alpha/alpha-particle-filtering.adoc

```diff
@@ -0,0 +1,99 @@
+[[algorithms-particle-filtering-centrality]]
+[.alpha]
+= Particle Filtering
+
+[abstract]
+--
+This section describes the Particle Filtering algorithm in the Neo4j Graph Data Science library.
+--
+
+Particle filtering measures the proximity between source nodes and other nodes in the graph.
+It is an alternative to the <<algorithms-page-rank-examples-personalised, Personalized Page Rank>> algorithm that runs directly against Neo4j rather than a projected graph.
+
+include::alpha-note.adoc[]
+
+* <<algorithms-particle-filtering-context, History and explanation>>
+* <<algorithms-particle-filtering-usecase, Use-cases - when to use the Particle Filtering algorithm>>
+* <<algorithms-particle-filtering-syntax, Syntax>>
+* <<algorithms-particle-filtering-examples, Examples>>
+
+[[algorithms-particle-filtering-context]]
+== History and explanation
+
+Particle Filtering was proposed by Denis Gallo, Matteo Lissandrini, and Yannis Velegrakis in their 2020 paper https://people.cs.aau.dk/~matteo/pdf/EDBT20-particle-filtering.pdf[Personalized Page Rank on Knowledge Graphs: Particle Filtering is all you need!^].
```

Comment:
The full reference for the core Particle Filtering Idea is:

> Lao, Ni, and William W. Cohen. "Fast query execution for retrieval models based on path-constrained random walks." Proceedings of the 16th ACM SIGKDD international conference on Knowledge discovery and data mining. 2010.
> https://doi.org/10.1145/1835804.1835916

While full reference the specific algorithm refers to this

> Gallo, Denis; Lissandrini, Matteo; and Velegrakis, Yannis.
“Personalized Page Rank on Knowledge Graphs: Particle Filtering is all you need!.”
Proceedings of the 23th International Conference on Extending Database Technology, EDBT 2020 (169-180). 
> http://dx.doi.org/10.5441/002/edbt.2020.54